### ISSUE TITLE
does anyone use this? 

### ISSUE THREAD

**Issue by jquense** [css-modules/postcss-icss-selectors#126]
This plugin seems like it was replaced or superseded by https://github.com/css-modules/postcss-modules-scope?

**Comment by alexander-akait** [comment]
@jquense No, it was excremental plugins and were not integrated to `css-loader` or postcss plugin, so it is abandoned, avoid using this repositories
